openslides/config/serializers.py

```diff
@@ -0,0 +1,9 @@
+from openslides.utils.rest_api import serializers, VariableField
+
+
+class ConfigSerializer(serializers.Serializer):
```

Comment:
I don't see why we need an extra serializer here. Config list, (retrieve) and update can be easily done in the ConfigViewSet and the cached data are still in a nice format.